.github/workflows/bake.yaml

```
@@ -78,24 +132,54 @@ jobs:
-      - name: Login to GCP
+      - name: 🔐 Authenticate — Google Cloud
         if: ${{ fromJSON(inputs.gcr) }}
-        id: auth
+        id: gcp-auth
         uses: google-github-actions/auth@v2
         with:
           credentials_json: "${{ secrets.GCP_CREDENTIALS_JSON }}"
 
-      - name: authenticate to GCR
+      - name: 🔐 Configure docker for GAR
         if: ${{ fromJSON(inputs.gcr) }}
```

actionlint (1.7.12)

[error] 1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)